kernel/src/process.rs

```diff
@@ -916,8 +929,10 @@ impl<'a> Process<'a> {
               \r\n  R4 : {:#010X}    R11: {:#010X}\
               \r\n  R5 : {:#010X}    R12: {:#010X}\
               \r\n  R9 : {:#010X} (Static Base Register)\
-              \r\n  PC : {:#010X} [{:#010X} in lst file]\
+              \r\n  SP : {:#010X} (Process Stack Pointer)\
               \r\n  LR : {:#010X} [{:#010X} in lst file]\
+              \r\n  PC : {:#010X} [{:#010X} in lst file]\
+              \r\n YPC : {:#010X} [{:#010X} in lst file]\
```

Comment:
Why print both? Are they both ever meaningful at the same time? How does an app developer know what the difference is? How does an app developer even know what "YPC" means?